code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -6,15 +6,13 @@
 	species_traits = list(MUTCOLORS, NO_UNDERWEAR, HAS_FLESH, HAS_BONE)
 	inherent_traits = list(
 		TRAIT_ADVANCEDTOOLUSER,
-		TRAIT_CAN_STRIP,
-		TRAIT_NOSLIPALL,
+		TRAIT_CAN_STRIP
```

Comment:
```suggestion
		TRAIT_CAN_STRIP,
```

Keep the trailing comma, I put it there for just this reason!

Reply:
Oops, didn’t notice, will do